### PR TITLE
Add per logger colors

### PR DESCRIPTION
**Pull request by adamcik**

### CODE REVIEW

File: docs/config.rst [hunk 1]
```diff
@@ -128,6 +128,14 @@ Logging configuration
     The ``loglevels`` config section can be used to change the log level for
     specific parts of Mopidy during development or debugging. Each key in the
     config section should match the name of a logger. The value is the log
+    level to use for that logger, one of ``black``, ``red``, ``green``,
+    ``yellow``, ``blue``, ``magenta``, ``cyan`` or ``white``.
```
Comment (jodal) [inline docs/config.rst]: You've mixed up the list of log levels and the list of log colors.

File: mopidy/config/schemas.py [hunk 2]
```diff
@@ -94,17 +94,16 @@ def serialize(self, values, display=False):
         return result
 
 
-class LogLevelConfigSchema(object):
-    """Special cased schema for handling a config section with loglevels.
+class MapConfigSchema(object):
+    """Special cased schema for handling mulitple keys with the same type.
```
Comment (jodal) [inline mopidy/config/schemas.py]: s/mulitple/multiple/

Comment (jodal) [inline mopidy/config/schemas.py]: Maybe not very "Special cased" any more?

**Comment by jodal** [comment]
+1 with comments

**Comment by adamcik** [comment]
Fixed.

**Pull request merged**